### PR TITLE
- Update recording tags to be more consistent with Kodi

### PR DESCRIPTION
**Pull request by rbarthen**
- add episode number
  - add series number
  - add Episode name
  - remove episode name from primary series title
  - remove episode name from plot outline

### CODE REVIEW

**Review by phunkyfish — APPROVED** [review]
LGTM

File: pvr.argustv/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+v20.3.1
```
Comment (phunkyfish) [inline pvr.argustv/changelog.txt]: Sorry, I spoke too soon. As you are adding functionality the version number should `20.4.0`.  You would use `20.3.1` if you had fixed a bug.

**Review by phunkyfish — CHANGES_REQUESTED** [review]

File: pvr.argustv/addon.xml.in [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.argustv"
-  version="20.3.0"
+  version="20.3.1"
```
Comment (phunkyfish) [inline pvr.argustv/addon.xml.in]: Sorry, I spoke too soon. As you are adding functionality the version number should 20.4.0. You would use 20.3.1 if you had fixed a bug.

Comment (ksooo) [reply pvr.argustv/addon.xml.in]: 20., not 2. ;-)

File: pvr.argustv/changelog.txt (hunk 1 above)
Comment (ksooo) [reply pvr.argustv/changelog.txt]: 20., not 2. ;-)

File: pvr.argustv/addon.xml.in (hunk 2 above)
Comment (phunkyfish) [reply pvr.argustv/addon.xml.in]: Whoops, fixed.

File: pvr.argustv/changelog.txt (hunk 1 above)
Comment (phunkyfish) [reply pvr.argustv/changelog.txt]: Whoops, fixed

File: pvr.argustv/addon.xml.in (hunk 2 above)
Comment (rbarthen) [reply pvr.argustv/addon.xml.in]: @phunkyfish 
I thought of this is a bug fix. It seemed to be a misconfiguration.  I was torn though about it being added functionality vs a bug fix though.  I can change it.  Are you sure you wouldn't consider it new functionality?

Comment (ksooo) [reply pvr.argustv/addon.xml.in]: It's new functionality. There is no misbehavior that your PR fixes. This, please update the add-on minor version => 20.4.0

**Review by phunkyfish — APPROVED** [review]

**Comment by phunkyfish** [comment]
Thanks again for the contributions.

**Pull request merged**